cmd/backup/backup.go

```diff
@@ -72,11 +72,11 @@ func (c *command) backup(savePath string, out io.Writer) error {
 	}
 
 	if savePath != "-" && !dir.IsDirectory(savePath) {
-		return fmt.Errorf("the save-path directory (%v) does not exist", savePath)
+		return fmt.Errorf("the save-path directory (%s) does not exist", savePath)
```

Comment:
`%s` should be fine for strings or stringers.

Reply:
of course the `String()` could panic if it's a `fmt.Stringer` so `%v` is safer.

For paths and things like that `%q` might be better to get a quoted string.

Reply:
> of course the String() could panic if it's a fmt.Stringer so %v is safer.

That's true. If String() is expected to panic, I agree that %v is the right choice. As for unexpected panics: Those are handled within the printf implementation anyways and won't reach the caller.

> For paths and things like that %q might be better to get a quoted string.

Also agree! Didn't want to change the error/log output too much in this PR, though. This might get into another dedicated PR.